sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -246,7 +260,86 @@ func (m *handlerImpl) processRerunScheduledScanRequest(requestID string, request
 	}
 	return m.composeAndSendApplyScanConfigResponse(requestID, err)
 }
+func (m *handlerImpl) processScanConfigScheduleChangeRequest(requestID string, config scanScheduleConfiguration) bool {
```

Comment:
Is the idea to use this method when we get around to implementing the update flow as well?  I see schedule is referenced in here, but does not seem to be set or used

Reply:
I think the schedule is updated at https://github.com/stackrox/stackrox/pull/9005/files#diff-934acc4d45abf0e86d3d3656f7dc018e3af42951927a56ddd8815b1d58c6f9a0R303

Reply:
I wonder if there is anything I missed